sql/moz-fx-data-shared-prod/contextual_services/dataset_metadata.yaml

```
@@ -0,0 +1,10 @@
+friendly_name: contextual_services
+description: |-
+  Derived data related to Contextual Services
```

```suggestion
  Data related to Contextual Services
```

Maybe these should just match the default descriptions that include the link to generated-schemas.

---

I've changed these to match the default descriptions in most cases. I kept Firefox accounts as-is, since most of the data presented there is not actually from the `firefox-accounts` namespace in the pipeline, but is instead from Cloud Logging.